control-plane-operator/controllers/hostedcontrolplane/konnectivity/params.go

```diff
@@ -49,11 +49,42 @@ func NewKonnectivityParams(hcp *hyperv1.HostedControlPlane, releaseImageProvider
 					Path:   "healthz",
 				},
 			},
-			InitialDelaySeconds: 120,
-			TimeoutSeconds:      30,
-			PeriodSeconds:       60,
-			FailureThreshold:    3,
-			SuccessThreshold:    1,
+			TimeoutSeconds:   5,
```

Comment:
from @Tamas-Biro1 - 

Konnectivity introduced a smarter readiness check https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/485 and we tried to do better on our side. We replaced the initialDelaySeconds because it does not have positive effect, it just a static wait before performing the first probe. The startupProbe replaces liveness/readiness probes during startup and ensures the container is fully started.
Regarding the others; we calculated farily good intervals which better align with Konnectivity (based on our previous measurements):
the periodSeconds set to 30 for both livenessProbe and readinessProbe and the failureThreshold is 6 and 1, so the waiting time is =>
30*6=180 seconds for livenessProbe
and 30*1=30 seconds for readinessProbe.